tests/h2-tests/tests/server.rs

```diff
@@ -1214,7 +1214,12 @@ async fn extended_connect_protocol_enabled_during_handshake() {
 
         let mut srv = builder.handshake::<_, Bytes>(io).await.expect("handshake");
 
-        let (_req, mut stream) = srv.next().await.unwrap().unwrap();
+        let (req, mut stream) = srv.next().await.unwrap().unwrap();
+
+        assert_eq!(
```

Comment:
Just to confirm, this assertion fails without the above change, right?

Reply:
Yes, sorry, this would fail.